src/optionals/optionals.h

```diff
@@ -12,7 +12,6 @@
 #include "log.h"
 #include "http/http.h"
 #include "path.h"
-#include "c.h"
```

Comment:
Not sure if this was intentional but we can probably remove the C module in it's entirety now to be honest. Can be done as a diff PR tho I guess.

Thanks for this!

Reply:
It was included twice so I figured I'd remove one. I can get a pr in to remove the c module soon.